data/journeys/journeys.md

```diff
@@ -27,4 +27,49 @@ Milestone: Scale
   Major action
   Major action
   ...
-```
\ No newline at end of file
+```
+
+## New Ansible user
+
+```yaml
+Milestone: Learn about Ansible
+  Read a brief description of what Ansible is
+  Redirect to the docs for onward learning journey, if needed
+Milestone: Install Ansible
+  Learn how to install via pip
+  Find out if OS-level packages exist for a system
+Milestone: Get started with simple executions
+  Redirect to the docs for onward quick-start journey
+Milestone: Ask a question
+  Understand what mediums are available
+  Choose a preferred medium, and redirect appropriately
+  Ask the question
+```
+
+## New Community member
```

Comment:
New contributor?

Reply:
If that encompasses forum support, chat, etc, then yes. Obviously it does to us, I'm asking from a common-arlance view....